internal/k8s/probe.go

```
@@ -71,6 +73,40 @@ func (pm *ProbeManager) ValidateURL(targetURL string) error {
+
+	probeURL := fmt.Sprintf("http://%s/probe?target=%s&module=%s",
+		config.ProberURL,
+		url.QueryEscape(targetURL),
+		url.QueryEscape(config.Module))
+
+	resp, err := pm.httpClient.Get(probeURL)
+	if err != nil {
+		logger.Debugf("Pre-flight check failed for %s: %v", targetURL, err)
+		return false
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)